src/bindings/capi/thorvg_capi.h

```diff
@@ -813,13 +813,14 @@ TVG_EXPORT Tvg_Paint* tvg_paint_duplicate(Tvg_Paint* paint);
 
 
 /*!
-* \brief Gets the bounding box of the Tvg_Paint object before any transformation.
+* \brief Gets the bounding box of the Tvg_Paint object before any transformation. (BETA_API)
```

Comment:
#868 :
``` 
* \brief Gets the axis-aligned bounding box of the paint object. (BETA_API)
*
* In case \p transform is \c true, all object's transformations are applied first, and then the bounding box is established. Otherwise, the bounding box is determined before any transformations.
```

Reply:
I'd add somewhere the more detailed description, but I see, that u don't :) so as you wish